data_structures/src/error.rs

```diff
@@ -262,10 +262,14 @@ pub enum TransactionError {
     ValueTransferWeightLimitExceeded { weight: u32, max_weight: u32 },
     /// Data Request weight limit exceeded
     #[fail(
-        display = "Data Request Transaction weight ({}) exceeds the limit ({})",
-        weight, max_weight
+        display = "Data Request Transaction ({:?}) weight ({}) exceeds the limit ({})",
```

Comment:
Just asking myself if there is any cleaner way of identifying the DR transaction... `dr_output` in Debug might be ugly.

Reply:
The obvious way is the transaction hash

Reply:
but that error is also used in the creation of a DataRequestTransaction, that could be achieved before to exist the proper transaction. Maybe I could include the witnesses number, because is the field that more affects to the weight

Reply:
For me it is prrety clear, as this error jumps  when the DR TX is being created. If it cannot be created, I understand that we need to log the body, as the tx hash will not point to anything

Reply:
or is this error also used when validating a dr tx?